### DISCUSSION TITLE
Drone life span

### DISCUSSION THREAD

**Discussion by Kalib-Drift** [bitcraze/discussions#536] [Q&A] [answered]
Hey everyone, I had a quick question about the drone lifespan. this is an important thing for me to think about when looking at how often I should expect to replace drones.

assuming minimal crashing what is your expected lifespan of a crazyflie 2.1?

with use during development (substantial crashing) how long would you expect the crazyflie to last? 

assuming similar use and troubleshooting as your big show before the end of the year how long would you expect those drones to last?

**Comment by tobbeanton** [comment, marked as answer]
These are difficult questions and I don't think I will be able to give you more then a very vague answers.

> Assuming minimal crashing what is your expected lifespan of a crazyflie 2.1?
Battery and motors would run out first. The battery general performance is specified to have more the 150 cycles with at least 80% charge left (2C charge and 10C discharge = Normal crazyflie usage).

Motors are harder and we don't have any long term general performance figures. A guess is around 200 hours of normal usage before brushes wear out.

> with use during development (substantial crashing) how long would you expect the crazyflie to last?

It is very seldom something crucial breaks, but spare parts like motor mounts, propellers and motors will be consumed. Most of our prototypes that are many years old still works fine.

> assuming similar use and troubleshooting as your big show before the end of the year how long would you expect those drones to last?

During our test show we broke one Crazyflie beyond repair. Spare parts we more used, maybe ~2 motors, ~10 motor mounts and ~4 propellers. If they drop out from 5-10m  usually at least one motor mount breaks. It might be wise to design some form of e.g. 3D printed protection for them. That could improve robustness and safety. Giving some form of lifetime number is too hard for me to even guess, too many variables...

**Reply by Kalib-Drift** [reply]
Alright thank you Tobias.
